mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/Rx.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.mesos.rx.java;
+
+import org.jetbrains.annotations.NotNull;
+import rx.Scheduler;
+import rx.schedulers.Schedulers;
+
+public final class Rx {
```

Comment:
Since this just delegates to `Schedulers`, I assume this is here so that we can change the schedulers that our library uses by updating this file only?